tests/phpunit/tests/blocks/updateIgnoredHookedBlocksPostMeta.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * Tests for update_ignored_hooked_blocks_postmeta
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ *
+ * @since 6.6.0
+ *
+ * @group blocks
```

Comment:
I saw the other day that for a test class where all methods cover the same function, we can do

```suggestion
 * @group blocks
 * @covers ::update_ignored_hooked_blocks_postmeta
```

and remove it from the individual methods 😄

Reply:
Fixed by @tjcafferkey in 79b9bedc22c424744a239af040181ea911936d6c.